src/test/test-execute.c

```diff
@@ -43,7 +43,8 @@ static bool can_unshare;
 
 STATIC_DESTRUCTOR_REGISTER(user_runtime_unit_dir, freep);
 
-typedef void (*test_function_t)(Manager *m);
+/* Return the number of services spawned */
+typedef int (*test_function_t)(Manager *m);
```

Comment:
Counting tests in such a way becomes easily outdated.
Instead, please introduce a global counter and increment it in `_test()`, and reset it in `run_tests()`.

Reply:
ok, done